api/v1beta1/openstacknetconfig_types.go

```diff
@@ -104,6 +104,12 @@ type Network struct {
 	// +kubebuilder:default=true
 	// VIP create virtual ip on the network
 	VIP bool `json:"vip"`
+
+	/*
+		// +kubebuilder:validation:Optional
+		// StaticReservations, manual/static IP address reservations per node
+		StaticReservations map[string]OpenStackMACNodeReservation `json:"staticReservations"`
```

Comment:
should this be removed?

Reply:
we could, but want to work on the static ip reservations as one of my next things and kept it as a reminder :)